spec/cqm/models_spec.rb

```diff
@@ -26,18 +47,25 @@
 
   it 'patients have patient characteristics' do
     expect(@patient_a.birth_datetime).to be
-    expect(@patient_b.birth_datetime).to be
+    expect(@patient_b.given_names).to be
```

Comment:
Des `expect(...).to be` in `rspec` just mean "I expect this to exist"?

Reply:
Yes, indeed!